src/utilities.client.js

```diff
@@ -399,6 +400,11 @@ function validateProxyField(fieldKey, value, isRequired = false, options = null)
     // If Apify proxy is not used skip additional checks
     if (!useApifyProxy) return fieldErrors;
 
+    // If Apify proxy is used, check if there is a selected country and if so, check that it's valid (empty or a valid country code)
+    if (apifyProxyCountry && apifyProxyCountry !== '' && !countries[apifyProxyCountry]) {
```

Comment:
The `apifyProxyCountry !== ''` condition is redundant, the first one will cover this.

Reply:
Fixed in fdb4a6c.